crates/pop-parachains/src/deployer_providers.rs

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use std::time::{SystemTime, UNIX_EPOCH};
+use strum::{EnumMessage as _, EnumProperty as _};
+use strum_macros::{AsRefStr, Display, EnumMessage, EnumProperty, EnumString, VariantArray};
+
+/// Supported deployment providers.
+#[derive(
+	AsRefStr,
+	Clone,
+	Debug,
+	Display,
+	EnumMessage,
+	EnumString,
+	EnumProperty,
+	Eq,
+	PartialEq,
+	VariantArray,
+)]
+pub enum DeploymentProvider {
+	/// Polkadot Deployment Portal (PDP). This provider enables seamless deployment of Polkadot
+	/// Native Rollups through the Polkadot Cloud.
+	#[strum(
+		ascii_case_insensitive,
+		serialize = "pdp",
+		message = "Polkadot Deployment Portal",
+		detailed_message = "Effortlessly deploy Polkadot Native Rollups on the Polkadot Cloud",
+		props(
+			BaseURL = "https://staging.deploypolkadot.xyz",
```

Comment:
Before merge to main, needs to be change to https://www.deploypolkadot.xyz/

Reply:
Staging will be the url for the following months: https://staging.deploypolkadot.xyz